lib/active_record/tenanted/database_adapter.rb

```diff
@@ -25,6 +25,8 @@ def new(db_config)
       end
 
       register "sqlite3", "ActiveRecord::Tenanted::DatabaseAdapters::SQLite"
+      register "trilogy", "ActiveRecord::Tenanted::DatabaseAdapters::MySQL"
```

Comment:
Eventually I'd like to run the mysql tests against trilogy. Doesn't need to be in this PR, though.

Reply:
Agree! Right now we're using mysql2 in our app but we should be switching over to trilogy this year / early next. So I'll also be able to test it out IRL too.